drivers/sensor/stmemsc/stmemsc_spi.c

```diff
@@ -11,6 +11,9 @@
 
 #define SPI_READ		(1 << 7)
 
+ /* Enable address auto-increment on some stmemsc sensors */
+#define STMEMSC_SPI_ADDR_AUTO_INCR		0x40
```

Comment:
you could probably use `BIT()` macros for these two, whatever is easier to cross reference with the datasheet I guess, same in the i2c file

Reply:
done, thx!